src/recommenderapp/templates/landing_page.html

```diff
@@ -1,54 +1,27 @@
-    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p" crossorigin="anonymous"></script>
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-  </head>
+{% extends 'shared/layout.html' %}
 
-  <body class="landing-page"> 
-    <nav class="navbar navbar-expand-lg navbar-dark bg-dark topNavBar fixed-top" id="landingTopNav">
-      <div class="container-fluid">
-        <a class="navbar-brand" href="#">PopcornPicks🍿</a>
-        <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
-        </button>
-      </div>
-    </nav>
-
-    <div class="container" style="margin-top: 60px;" id="centralDivLanding">
-      <div class="heading1"><br><br><br><br>
-        <h2><center>🎬 PopcornPicks🍿: Pick a Movie! 🎬</center></h2>
-        <p style="text-align: center; color: azure; font-size: 18px;"><br>
-        Discover personalized movie recommendations by selecting up to 5 of your favorite films.
-        <br> 
-        Create a watchlist and have it conveniently sent to your email.
-        <br>
-        Enjoy movies at your own pace, on your terms.
-        </p><br>
-      <button id="getStartedButton" onclick="getStarted()" class="btn btn-primary mx-auto">Get Started!</button><br><br>
-        <div class="highlighted-section">
-          <p>Made with ❤️ by <a href="https://github.com/adipai/PopcornPicks" target="_blank"> PopcornPicks</a></p>     
-          <a href="https://github.com/adipai/PopcornPicks/blob/master/LICENSE.md" target="_blank">MIT License Copyright (c) 2023 PopcornPicks</a>
-        </div>
-      </div>
-    </div>
-    <div class="d-none" id="loaderLanding">
-      <div class="spinner-border" role="status">
-        <span class="sr-only"></span>
-      </div>
+{% block maincontent %}
+<div class="container" style="margin-top: 60px;" id="centralDivLanding">
+  <div class="heading1"><br><br><br><br>
+    <h2><center>🎬 PopcornPicks🍿: Pick a Movie! 🎬</center></h2>
+    <p style="text-align: center; color: azure; font-size: 18px;"><br>
+    Discover personalized movie recommendations by selecting up to 5 of your favorite films.
+    <br>
+    Create a watchlist and have it conveniently sent to your email.
+    <br>
+    Enjoy movies at your own pace, on your terms.
+    </p><br>
+  <button id="getStartedButton" onclick="getStarted()" class="btn btn-primary mx-auto">Get Started!</button><br><br>
+    <div class="highlighted-section">
+      <p>Made with ❤️ by <a href="https://github.com/adipai/PopcornPicks" target="_blank"> PopcornPicks</a></p>
```

Comment:
I think this URL will be pointing to our repo now?

Reply:
Fixed